contracts/lib/misc/TimeHelpers.sol

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-License-Identitifer:    MIT
+ */
+
+pragma solidity ^0.4.18;
+
+
+contract TimeHelpers {
```

Comment:
Last thing 😅, I'd move this to `contracts/common` since it's not imported / copied from somewhere else (it's similar to `IsContract`, which also lives there).